app/views/publish_document/published.html.erb

```diff
@@ -17,19 +17,20 @@
       <%= render "govuk_publishing_components/components/govspeak" do %>
         <%= govspeak_to_html t("publish_document.published.reviewed.body_govspeak", title: @document.title) %>
 
-        <%= link_to(nil, DocumentUrl.new(@document).public_url, class: "govuk-link") %>
+        <%= link_to(nil, DocumentUrl.new(@document).public_url, class: "govuk-link govuk-link--no-visited-state") %>
       <% end %>
     <% else %>
       <%= render "govuk_publishing_components/components/panel", {
         title: t("publish_document.published.published_without_review.title")
       } %>
 
       <%= render "govuk_publishing_components/components/govspeak" do %>
-        <%= govspeak_to_html t("publish_document.published.published_without_review.body_govspeak", title: @document.title) %>
+        <%= govspeak_to_html t("publish_document.published.published_without_review.body_govspeak", title: @document.title, link_to_live: DocumentUrl.new(@document).public_url) %>
 
-        <p> <%= t("publish_document.published.published_without_review.send_label") %> </p>
-
-        <%= link_to(nil, document_url(@document, utm_content: "2i-link"), class: "govuk-link") %>
+        <%= render "govuk_publishing_components/components/copy_to_clipboard",
```

Comment:
I think this should be outside of the govspeak component.

Reply:
👍